Package.swift

```diff
@@ -2,7 +2,7 @@
 import PackageDescription
 
 let package = Package(
-    name: "swift-baggage-context",
+    name: "swift-context",
```

Comment:
We are working on securing the namespace to move this repository to it's final destination.